index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="snapshot">
+        <img class="profile_pic" src="./images/TimScott.jpg" alt="Tim Scott" />
+        <div class="name">Tim Scott</div>
+        <div class="title">Senate (R-SC 1st District) - 2014</div>
+        <div class="h-line"></div>
+        <div class="funds">
+          <div class="number">$82,967</div>
+          <div class="type">Raised</div>
+        </div>
+        <div class="donors">
+          <div class="number">9,203</div>
+          <div class="type">Donors</div>
+        </div>
+      </div>
+      <div class="snapshot">
+        <img class="profile_pic" src="./images/MarkSanford.jpg" alt="Mark Sanford" />
+        <div class="name">Mark Sanford</div>
+        <div class="title">House  (R-SC 1st District) - 2014</div>
+        <div class="h-line"></div>
```

Comment:
instead of creating an empty HTML element, you could either
* create a wrapper around the bottom 2 elements and give it a `border-top`
* add border top to funds and donors